johnsonandjohnson/Bodiless-JS
# chore: Create gatsby starter.

## Changes
- Update minimal starter so it can deploy to gatsby cloud

## Test Instructions
- Clone repo and checkout and setup this branch
- From bodiless-cli directory
  ```
  npm run bl new -- --clone-local -r HEAD --setup "npm run setup:gatsby-cloud" --name minimal
  ```
- Push the new site to a github repository in a separate (non-jnj) account --not to the bodiless repo
- Sign up for free tier gatsby cloud account
- Create a new gatsby cloud project linked to the github repo -- set gatsby root dir to /sites/minimal.
- Verify that new site deploys correctly to gatsby cloud
- Aso verify that site builds and starts locally.

## Related Issues
Closes #1643
